scripts/lstbin_simple.py

```diff
@@ -60,6 +60,8 @@
 
 kwargs['output_flagged'] = not kwargs.pop('no_flagged_mode')
 kwargs['output_inpainted'] = kwargs.pop("do_inpaint_mode")
+kwargs['sigma_clip_subbands'] = [int(b) for b in kwargs["sigma_clip_subbands"].split(",")]
```

Comment:
Has this been added to the argparser?

Also, this raises an interesting question about the best way to specify subbands from the commandline and/or from a TOML. 

I'm using this format, which is in MHz. It's similar to something Aaron EW did for H4C, though I think he used channels. It's a lot more human readable, but it's a bit clunky:

https://github.com/HERA-Team/hera_pipelines/blob/46a6018bf497d1066fd242aaf01fa6e0a2aaa9fd/pipelines/h6c/idr2/v2/post-lstbin/h6c_pspec.toml#L15

Reply:
Yes, it's been added to the argparser. The format I'm using is like:

```
--sigma-clip-subbands 0,150,1000
```

This assumes that *every* channel belongs to a subband, so you just specify the edges. I could make it more general, using the tilde notation, but I'm not perfectly sure it's necessary in this case?

Reply:
I'd like to use the tilde notation. It's more standard and it's not necessarily true that every channel belongs to a subband (e.g. FM).

Reply:
OK, I've done this now.